### PR TITLE
Rename Windows images so that a standard DockerHub repo is used

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Renaming the image as discussed in https://groups.google.com/forum/#!topic/jenkinsci-dev/oxD-Hd_7l9k .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ To run a Docker container with [Work Directory](https://github.com/jenkinsci/rem
     
   Windows agent:
 
-    docker run jenkins/jnlp-agent-windows -Url http://jenkins-server:port -WorkDir=C:/Jenkins/agent -Secret <secret> -Name <agent name>
+    docker run jenkins/inbound-agent:latest-windows -Url http://jenkins-server:port -WorkDir=C:/Jenkins/agent -Secret <secret> -Name <agent name>
```
Comment (slide) [inline README.md:38]: I think we can just say jenkins/inbound-agent:latest here

Comment (oleg-nenashev) [reply README.md:38]: No, it is an example for Windows

Comment (oleg-nenashev) [reply README.md:38]: OK, now I see what's the problem. We have a naming collision between `latest` in the baseline and `latest` in Windows scripts. So we will get into https://github.com/jenkinsci/docker-agent/pull/126 equivalent if it is merged as is

```suggestion
    docker run jenkins/inbound-agent:latest-windows -Url http://jenkins-server:port -WorkDir=C:/Jenkins/agent -Secret <secret> -Name <agent name>
```

**Comment by oleg-nenashev** [comment]
@slide @timja @MarkEWaite 

My proposal is to use the following tags for windows:

* `jenkins/jnlp-agent:windows` - equivalent of the current `latest` in jenkins/jnlp-agent
* `windows-jdk8` - equivalent of the current `jenkins/jnlp-agent:jdk8`
* `windows-jdk11` - equivalent of the current `jenkins/jnlp-agent:jdk11`

I will not merge this PR for now, we can integrate it after `4.3-2` and then announce Windows images as official ones

**Comment by slide** [comment]
It should just show up as a different architecture for latest, other repos don't name windows images differently. The AdoptOpenJDK latest works for Linux and windows.

**Comment by slide** [comment]
See here https://hub.docker.com/_/adoptopenjdk?tab=tags

The latest tag has a Linux/amd64 and a windows/amd64 variant, but both are using the latest tag.

**Comment by slide** [comment]
Looks like we would need to use manifests to do it, so name them however you want for now. I would like to give all users the same experience (e.g., docker pull jenkins/agent) regardless of the platform, but that will have to wait I guess.

**Comment by oleg-nenashev** [comment]
> Looks like we would need to use manifests to do it, so name them however you want for now. I would like to give all users the same experience (e.g., docker pull jenkins/agent) regardless of the platform, but that will have to wait I guess

Yes, there is item for it in the roadmap. Just another reason to consoludate the build flow on Jenkins so that we can manage that

image

**Comment by oleg-nenashev** [comment]
> My proposal is to use the following tags for windows:

Postponing it to the next release. I think we could do it next week and then announce general availability of Windows images for agents

File: README.md (hunk 1 above)
Comment (slide) [reply README.md:38]: I don't think we are pushing a latest-windows right now. We probably want to use something else here:
```suggestion
    docker run jenkins/inbound-agent:windowsservercore-1809 -Url http://jenkins-server:port -WorkDir=C:/Jenkins/agent -Secret <secret> -Name <agent name>
```

**Pull request merged**